text/0000-project-goals.md

```diff
@@ -0,0 +1,205 @@
+    * they need to decide in a timely fashion whether they can commit the proposed resources
+
+## Outputs and milestones
+
+### Outputs
+
+There are three specific outputs from this process:
+
+* A **goal slate for the second half (H2) of 2024**, which will include
+    * a set of goals, each with an owner and with approval from their associated teams
+    * a high-level write-up of why this particular set of goals was chosen and what impact we expect for Rust
+    * plan is to start with a smallish set of goals, though we don't have a precise number in mind
+* **Regular reporting** on the progress towards these goals over the course of the year
+    * monthly updates on Inside Rust (likely) generated by scraping tracking issues established for each goal
+    * larger, hand authored updates on the main Rust blog, one in October and a final retrospective in December
+* A **goal slate for the first half (H1) of 2025**, which will include
+    * a set of goals, each with an owner and with approval from their associated teams
+    * a high-level write-up of why this particular set of goals was chosen and what impact we expect for Rust
+    * (probably) a goal to **author an RFC with a finalized process** that we can use going forward
+
+### Milestones
+
+Key milestones along the way (with the most impactful highlighted in bold):
+
+| Date         | 2024 H2 Milestone                                               | 2025 H1 Milestones                                                         |
+| ------------ | --------------------------------------------------------------- | -------------------------------------------------------------------------- |
+| **Apr 26**   | **Kick off the goal collection process**                        |                                                                            |
+| May 24       | Publish draft goal slate, take feedback from teams              |                                                                            |
+| June 14      | Approval process for goal slate begins                          |                                                                            |
+| **June 28**  | **Publish final goal slate**                                    |                                                                            |
+| July         | Publish monthly update on Inside Rust                           |                                                                            |
+| August       | Publish monthly update on Inside Rust                           |                                                                            |
+| September    | Publish monthly update on Inside Rust                           |                                                                            |
+| **Oct**      | **Publish intermediate goal progress update on main Rust blog** | **Begin next round of goal process, expected to cover first half of 2025** |
```

Comment:
It's probably good to set some dates here now, because people tend to disappear in the second half of December, and we don't want to wait too long to kick off the goal selection process.

Reply:
Suggestion: Blog post Oct 1, kick off next round Oct 7?

Reply:
OK

Reply:
I just put Oct 1 for both